pkg/installer/database_objects.go

```diff
@@ -6,7 +6,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.2.8
```

Comment:
We should move this to use the new embed package at some point

Reply:
@laverya I understand the comment, but not sure about the value yet. To make sure this doesn't get lost, would you be up for creating an issue that explains the value of the change?